recover/recovery.go

```diff
@@ -85,7 +85,7 @@ func (r *recoverer) WalletFromSharedSecret(pk *mcrypto.PrivateKeyInfo) (mcrypto.
 	}
 
 	kp := mcrypto.NewPrivateKeyPair(sk, vk)
-	return monero.CreateWallet("recovered-wallet", r.env, r.xmrClient, kp)
+	return monero.CreateWallet("recovered-wallet", r.env, r.xmrClient, kp, 0)
```

Comment:
We should revisit this when going over the recoverer. If we record the block height before the swap started in the recovery information, we could speed recovery wallet synchronization.